src/openforms/authentication/contrib/digid_eherkenning_oidc/views.py

```diff
@@ -43,6 +43,11 @@ def get(self, request):
         if not next_url:
             raise DisallowedRedirect
 
+        # This is needed for the next steps in order to redirect the user to the right place again.
+        # When a user is authenticated and cancels the login flow, the next url (oidc_login_next)
```

Comment:
IMO being logged out of the admin is not an issue - I also don't see it reported in the ticket description as being an issue, only the `KeyError` started showing up in Sentry because of information being lost.